proto/atomone/gov/v1/query.proto

```diff
@@ -61,6 +61,31 @@ service Query {
     option (google.api.http).get =
         "/atomone/gov/v1/proposals/{proposal_id}/tally";
   }
+
+  // Governor queries governor information based on governor address.
+  rpc Governor(QueryGovernorRequest) returns (QueryGovernorResponse) {
+    option (google.api.http).get = "/atomone/gov/v1/governor/{governor_address}";
+  }
+
+  // Governors queries all governors.
+  rpc Governors(QueryGovernorsRequest) returns (QueryGovernorsResponse) {
+    option (google.api.http).get = "/atomone/gov/v1/governors";
+  }
+
+  // GovernanceDelegations queries all delegations of a governor.
+  rpc GovernanceDelegations(QueryGovernanceDelegationsRequest) returns (QueryGovernanceDelegationsResponse) {
+    option (google.api.http).get = "/atomone/gov/v1/governors/{governor_address}/delegations";
+  }
+
+  // GovernanceDelegation queries a delegation
+  rpc GovernanceDelegation(QueryGovernanceDelegationRequest) returns (QueryGovernanceDelegationResponse) {
+    option (google.api.http).get = "/atomone/gov/v1/delegations/{delegator_address}";
```

Comment:
I would rather see singular `delegation` here, as it will always return a single.

Reply:
This is for consistency, the plural is used in other places too (e.g. proposals)

Reply:
Understandable, then the first one `/atomone/gov/v1/governor/{governor_address}` should be `/atomone/gov/v1/governors/{governor_address}` ?

Reply:
> Understandable, then the first one `/atomone/gov/v1/governor/{governor_address}` should be `/atomone/gov/v1/governors/{governor_address}` ?

yes